src/gt4py/next/ffront/foast_to_itir.py

```diff
@@ -132,6 +138,64 @@ def visit_BlockStmt(
         assert inner_expr
         return inner_expr
 
+    def visit_IfStmt(
+        self, node: foast.IfStmt, *, inner_expr: itir.Expr | None, **kwargs
```

Comment:
I think the exact meaning of `inner_expr` should be documented somewhere, because it is not so easy to understand completely by looking at the codebase for the first time.